package.json

```diff
@@ -30,6 +30,7 @@
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
     "babel-jest": "^23.6.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
```

Comment:
Why is this required? All of our supported browsers support the object spread syntax.

Reply:
Without it, Storybook was crashing as it didn't recognise the syntax.

Reply:
I'm not a fan of modifying our production code because of a separate tool... Maybe storybook requires this webpack 'hack'...

https://github.com/juju/juju-gui/blob/develop/package.json#L96

Reply:
Can you check to see if this change means that we do not require this?

Reply:
Negative. Throws the same error:

<img width="633" alt="screenshot 2018-11-30 at 10 27 32" src="https://user-images.githubusercontent.com/505570/49283855-cefab080-f48a-11e8-956c-b6ad03da3e5d.png">

Worth saying this isn't a community polyfill, it's a supported Babel extension - https://babeljs.io/docs/en/babel-plugin-proposal-object-rest-spread

Reply:
So after some further investigation it seems that babel itself doesn't support spread syntax even though all supported browsers and node do...so we have to transpile it for now...We'll just have to keep our eyes on it and remove this when we can update babel.